src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentBuilderWidget/PaymentBuilderWidget.tsx

```diff
@@ -92,6 +95,11 @@ const PaymentBuilderWidget: FC<PaymentBuilderWidgetProps> = ({ action }) => {
     stopPolling,
   } = useGetExpenditureData(expenditureId, { pollUntilUnmount: true });
 
+  const tokenData = getSelectedToken(
+    colony,
+    expenditure?.slots?.[0].payouts?.[0]?.tokenAddress ?? '',
```

Comment:
You're missing a `?` just before `.payouts`, which means the app can still crash if expenditure slots are not set yet:
<img width="501" alt="image" src="https://github.com/user-attachments/assets/161582c5-db16-42fd-b94f-b1055c3849bd" />